### DISCUSSION TITLE
Connection Error Out

### DISCUSSION THREAD

**Discussion by DarksealStudios** [volotat/SD-CN-Animation#58] [Q&A]
Traceback (most recent call last):
  File "F:\Ai\Automatic1111\stable-diffusion-webui\modules\scripts.py", line 444, in postprocess
    script.postprocess(p, processed, *script_args)
TypeError: Script.postprocess() missing 11 required positional arguments: 'is_enabled', 'codec', 'interpolation', 'duration', 'skip_steps', 'debug', 'run_incomplete', 'tmp_delete', 'out_create', 'tmp_path', and 'out_path'

Got to the last frame and error out, not saving the video.

**Comment by volotat** [comment]
Please update webui to the latest version as well as the 'sd-webui-controlnet' extension. Tell me if it helped.

**Reply by DarksealStudios** [reply]
I was using txt2video, not using controlnet, but I do have 1.1, updated on the 3rd. Will do though later...

**Reply by DarksealStudios** [reply]
also, do you know what update hash of automatic, or what requirement of the update is needed? I don't want to do a blind gitpull

**Reply by volotat** [reply]
This one should work https://github.com/AUTOMATIC1111/stable-diffusion-webui/releases/tag/v1.1.0

**Reply by DarksealStudios** [reply]
thank you very much but it's nothing but errors for me:

venv "F:\Ai\1111Automatic1.1\stable-diffusion-webui-1.1.0\venv\Scripts\Python.exe"
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Commit hash: <none>
Installing gfpgan
Traceback (most recent call last):
  File "F:\Ai\1111Automatic1.1\stable-diffusion-webui-1.1.0\launch.py", line 352, in <module>
    prepare_environment()
  File "F:\Ai\1111Automatic1.1\stable-diffusion-webui-1.1.0\launch.py", line 260, in prepare_environment
    run_pip(f"install {gfpgan_package}", "gfpgan")
  File "F:\Ai\1111Automatic1.1\stable-diffusion-webui-1.1.0\launch.py", line 128, in run_pip
    return run(f'"{python}" -m pip {command} --prefer-binary{index_url_line}', desc=f"Installing {desc}", errdesc=f"Couldn't install {desc}", live=live)
  File "F:\Ai\1111Automatic1.1\stable-diffusion-webui-1.1.0\launch.py", line 96, in run
    raise RuntimeError(message)
RuntimeError: Couldn't install gfpgan.
Command: "F:\Ai\1111Automatic1.1\stable-diffusion-webui-1.1.0\venv\Scripts\python.exe" -m pip install git+https://github.com/TencentARC/GFPGAN.git@8d2447a2d918f8eba5a4a01463fd48e45126a379 --prefer-binary
Error code: 1
stdout: Collecting git+https://github.com/TencentARC/GFPGAN.git@8d2447a2d918f8eba5a4a01463fd48e45126a379
  Cloning https://github.com/TencentARC/GFPGAN.git (to revision 8d2447a2d918f8eba5a4a01463fd48e45126a379) to c:\users\darkseal\appdata\local\temp\pip-req-build-drl9i_r9
  Resolved https://github.com/TencentARC/GFPGAN.git to commit 8d2447a2d918f8eba5a4a01463fd48e45126a379
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'error'

stderr:   Running command git clone --filter=blob:none --quiet https://github.com/TencentARC/GFPGAN.git 'C:\Users\Darkseal\AppData\Local\Temp\pip-req-build-drl9i_r9'
  Running command git rev-parse -q --verify 'sha^8d2447a2d918f8eba5a4a01463fd48e45126a379'
  Running command git fetch -q https://github.com/TencentARC/GFPGAN.git 8d2447a2d918f8eba5a4a01463fd48e45126a379
  Running command git checkout -q 8d2447a2d918f8eba5a4a01463fd48e45126a379
  error: subprocess-exited-with-error

  python setup.py egg_info did not run successfully.
  exit code: 1

  [1 lines of output]
  ERROR: Can not execute `setup.py` since setuptools is not available in the build environment.
  [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

Encountered error while generating package metadata.

See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
WARNING: There was an error checking the latest version of pip.


Press any key to continue . . .

**Reply by DarksealStudios** [reply]
When I tried a new install again it said:
venv "F:\Ai\1111Automatic1.1\stable-diffusion-webui-1.1.0\venv\Scripts\Python.exe"
The application has failed to start because its side-by-side configuration is incorrect. Please see the application event log or use the command-line sxstrace.exe tool for more detail.
Press any key to continue . . .

**Reply by volotat** [reply]
It might be some issue in the environment. Try to remove 'venv' folder and run webui again. It will reinstall all dependencies. It will take some time but should resolve the issue.

**Reply by DarksealStudios** [reply]
it was a brand new venv, you had just given it to me, it was in the process of making the venv when it did this. I'm just going to back away slowly and try again some other time. ty
